ui/lib/kv/addon/templates/secret/metadata/index.hbs

```diff
@@ -4,8 +4,14 @@
 ~}}
 
 <Page::Secret::Metadata::Details
+  @backend={{this.model.backend}}
+  @breadcrumbs={{this.breadcrumbs}}
+  @canDeleteMetadata={{this.model.permissions.metadata.canDelete}}
+  @canReadCustomMetadata={{or this.model.permissions.metadata.canRead this.model.permissions.data.canRead}}
```

Comment:
Again, not really sure why we need this capability since the action has already been taken

Reply:
Previously we couldn't just rely on `this.customMetadata` since we didn't know if no metadata meant we can't read it or it doesn't exist. Now that the permissions have moved into the route and the `@metadata` or `@secret` don't exist or return `null` (before `secret` always existed and metadata returned an empty object) we can solely rely on the getter `this.customMetadata`